examples/set-bucket-notification.js

```diff
@@ -21,14 +21,16 @@
 var Minio = require('minio')
 
 var s3Client = new Minio.Client({
-  endPoint: 's3.amazonaws.com',
+  endPoint: 'localhost',
+  port: 9000,
+  useSSL: false,
   accessKey: 'YOUR-ACCESSKEYID',
   secretKey: 'YOUR-SECRETACCESSKEY'
 })
 
 var config = new Minio.NotificationConfig()
-var arn = Minio.buildARN('aws', 'sns', 'us-east-1', 111112222233, 'topicresource')
-var topic = new Minio.TopicConfig(arn)
+var arn = Minio.buildARN('minio', 'sqs', '', 1, 'webhook')
+var topic = new Minio.QueueConfig(arn)
```

Comment:
@kannappanr , pls change the var topic -> queue here in the example

Reply:
done